### DISCUSSION TITLE
Welcome to clonesquad-ec2-pet-autoscaler Discussions!

### DISCUSSION THREAD

**Discussion by jcjorel** [jcjorel/clonesquad-ec2-pet-autoscaler#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by samiroun** [comment]
Hello @jcjorel ,

We are using Clonesquad and plan to update to the latest available release.

Would it be possible to inform us if a new release is planned in the next few days so that we can eventually enjoy the latest version?

Thanks in advance for your feedback

**Reply by jcjorel** [reply]
Hi,
No new release in coming days.
Next one will be to replace CloudWatch Events by EventBridge Scheduler to correctly support daylight saving time (it is forecasted as a change without any impact on existing configurations so a straight forward upgrade...)

**Reply by samiroun** [reply]
thank you for your feedback
